.github/workflows/python-tests.yml

```diff
@@ -86,66 +89,67 @@ jobs:
         run: echo "::set-output name=wheel_name::$(basename $(ls dist/*.whl))"
 
   os_test_matrix:
-      name: Python ${{ matrix.os }} Tests
```

Comment:
These spacing changes may cause pipeline issues (YAML cares about spacing). Can you please revert everything in this file that you didn't specifically mean to change so that it's clear what you did mean to change?